agentos/__init__.py

```diff
@@ -1,7 +1,5 @@
 """The ``agentos`` module provides an API for building learning agents."""
 
-from agentos.version import VERSION as __version__
```

Comment:
flake8 was complaining about this unused import, but is this something we need to make the package work right?